header injection modifications (#62)
mulesoft/docs-visualizer: modules/ROOT/pages/runtime-fabric-setup.adoc

== Enable Header Injection

> Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule 4.x released before April 7, 2020.

Reviewer: At least this one clarifies a little bit: RELEASED before
I still wonder why this is not a "version"....

Author: ```suggestion
Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule version 4.x released before April 7, 2020.
```

> Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule 4.x released before <insert RTF Mule Runtime Date with UA 2.2.2>.

Reviewer 2: Apr 7 2020

Author: ```suggestion
Anypoint Visualizer requires an additional header injection for Anypoint Runtime Fabric deployments. However, due to potential performance impact, header injection is disabled by default in Mule 4.x released before April 7, 2020.
```

* If you are deploying an app for the first time, navigate to the deployment page and click *Deploy application*.
. In the *Properties* tab, set the following property in the app: `anypoint.platform.config.analytics.agent.header_injection.disabled=false`

> Additionally, you must use version 1.5.12 or higher of the HTTP connector in your Mule 4 applications.

Reviewer: higher > later
HTTP Connector

Author: ```suggestion
Additionally, you must use version 1.5.12 or later of the HTTP Connector in your Mule 4 applications.
```
